repose-aggregator/components/services/datastore-service/datastore-service-impl/src/main/java/org/openrepose/core/services/datastore/impl/distributed/remote/command/AbstractRemoteCommand.java

```diff
@@ -64,7 +63,7 @@ public String getConnectionPoolId() {
     public abstract ServiceClientResponse execute(RequestProxyService proxyService, RemoteBehavior remoteBehavior);
 
     protected byte[] getBody() {
-        return null;
+        return new byte[0];
```

Comment:
yeah, i mentioned that on one of the other null cases. we need to check each of them carefully and check the contract where it's not our contract